### PR TITLE
Add Travis build status notification in HipChat

### PR DESCRIPTION
**Pull request by DaazKu**
Make Travis-CI send a notification in our HipChat server on build failure or when a build is fixed.

Fixes https://github.com/vanilla/vanilla/issues/5467

### CODE REVIEW

**Comment by DaazKu** [comment]
screen shot 2017-06-19 at 3 27 48 pm

Works well :)

File: .travis.yml [hunk 1]
```diff
@@ -40,3 +40,12 @@ script:
   - ls -lah ./conf
   - cat /tmp/error.log
   - cat /tmp/access.log
+
+notifications:
+  hipchat:
+    format: html
+    on_success: change
+    rooms:
+      - secure: "SsKmSAZFynBz4ZKm5NPyuXvNjIMyxpNMXsgfXVImG8xjQHdXjEpZAiyckK8E2lXBBypv59Oex6wsS0RvyxpI/mwQ9dTQ9ayurQxwH3V5Q/+pRbtXJOkP+DSIsHhRb9D4xa5nPbh4N48+QZvUFiH2ety9/gev4mtMkLv3lC0vgpc="
```
Comment (linc) [inline .travis.yml]: uhhhhh. wanna clarify why this is safe to be public?

Comment (DaazKu) [inline .travis.yml]: From: https://docs.travis-ci.com/user/notifications#Configuring-HipChat-notifications
screen shot 2017-06-29 at 10 48 56 am

See: https://docs.travis-ci.com/user/encryption-keys/

Comment (linc) [inline .travis.yml]: Thanks.

**Pull request merged**